spec/workers/adviser_sign_up_worker_spec.rb

```diff
@@ -0,0 +1,181 @@
+
+      it 'sends the international degree type' do
+        expect_sign_up({
+          degree_type_id: described_class::DEGREE_TYPES[:international],
+        })
+      end
+
+      it 'sends nil for uk_degree_grade_id if the grade is not recognised' do
+        degree.update(grade: '100%')
+        expect_sign_up({
+          degree_type_id: anything,
+          uk_degree_grade_id: nil,
+        })
+      end
+    end
+
+    describe 'sending GCSE qualification details' do
+      it "sends no for has_gcse_maths_and_english_id if they haven't got Maths or English GCSEs" do
+        application_form.maths_gcse.destroy
+        application_form.english_gcse.destroy
+
+        expect_sign_up({
+          has_gcse_maths_and_english_id: described_class::GCSE[:no],
+        })
+      end
+
+      it 'sends no for has_gcse_maths_and_english_id if they have passed Maths but not English GCSEs' do
+        application_form.english_gcse.update(grade: 'Z')
+
+        expect_sign_up({
+          has_gcse_maths_and_english_id: described_class::GCSE[:no],
+        })
+      end
+
+      it 'sends no for has_gcse_maths_and_english_id if they have passed English but not Maths GCSEs' do
+        application_form.maths_gcse.update(grade: 'Z')
+
+        expect_sign_up({
+          has_gcse_maths_and_english_id: described_class::GCSE[:no],
+        })
+      end
+
+      it "sends no for has_gcse_science_id if they haven't got a Science GCSE" do
+        application_form.science_gcse.destroy
+
+        expect_sign_up({
+          has_gcse_science_id: described_class::GCSE[:no],
+        })
+      end
+
+      it 'sends no for has_gcse_science_id if they have not passed their Science GCSE' do
+        application_form.science_gcse.update(grade: 'Z')
+
+        expect_sign_up({
+          has_gcse_science_id: described_class::GCSE[:no],
+        })
+      end
+
+      it 'sends yes for planning_to_retake_gcse_maths_and_english_id if they are completing both Maths and English GCSEs' do
+        application_form.maths_gcse.update(currently_completing_qualification: true)
+        application_form.english_gcse.update(currently_completing_qualification: true)
+
+        expect_sign_up({
+          planning_to_retake_gcse_maths_and_english_id: described_class::GCSE[:yes],
+        })
+      end
+
+      it 'sends yes for planning_to_retake_gcse_science_id if they are completing their Science GCSE' do
+        application_form.science_gcse.update(currently_completing_qualification: true)
+
+        expect_sign_up({
+          planning_to_retake_gcse_science_id: described_class::GCSE[:yes],
+        })
+      end
+    end
+  end
+
+  def expect_request_attribtues(attributes, expected_attributes)
+    expect(attributes).to include(expected_attributes)
+  end
+
+  def expect_sign_up(expected_attribute_overrides = {})
+    expect_any_instance_of(GetIntoTeachingApiClient::TeacherTrainingAdviserApi).to \
```

Comment:
Is there a way to avoid the backslash?  e.g. putting `.to received` on the newline